src/widgets/popOver.def.js

```diff
@@ -0,0 +1,183 @@
+import $ from 'jquery';
+import '../plugins/childOf';
+
+const doc = $(document),
+	win = $(window);
+
+export default {
+
+	options: {
+		target: '',
+		top: 0,
+		left: 0,
+		appendTo: 'body'
+	},
+
+	_create: function() {
+		const self = this,
+			o = self.options,
+			e = self.element;
+
+		self.showing = false;
+		
+		/****************************************************/
```

Comment:
Remove this comment block.  If the comment is important, consider extracting into a well named method or use a single block comment `/*  .... */` above it.  Also remove all names since that information will be saved in the commit.